docs/portal/developer-portal/uan_admin_guide.ditamap

```diff
@@ -32,7 +32,6 @@
     </topichead>
     <topichead>
         <topicmeta><navtitle>Advanced UAN Configuration</navtitle></topicmeta>
-        <topicref href="advanced/Customizing_UAN_Images_Manually.md" format="markdown"/>
```

Comment:
this also needs to be removed from `uan_install_guide.ditamap`.

Reply:
I don't see it in uan_install_guide.ditamap